sdk/program-pack/src/lib.rs

```diff
@@ -1,6 +1,13 @@
-//! The [`Pack`] serialization trait.
+//! The [`Pack`] serialization trait
+//! This is a Solana-specific serialization API that is used by many
```

Comment:
Hi @kevinheavey, one final comment on this. Perhaps we can tweak a bit the description here to read:
```
//! This is a specific serialization API that is used by many older programs in
//! the [Solana Program Library][spl] to manage account state. It is not generally 
//! recommended for new code since it does not define a language-independent
//! serialization format.
```